src/BaseTemplates/EmptyWeb/project.json

```diff
@@ -26,25 +26,22 @@
     "$netframeworkversion$": { }$endif$
   },
 
-  "compilationOptions": {
+  "buildOptions": {
     "emitEntryPoint": true,
     "preserveCompilationContext": true
   },
 $if$ ($context$ == WebCore)
   "runtimeOptions": {
-    "configProperties": {
-      "System.GC.Server": true
-    }
+    "gcServer": true
   },
 $endif$
-  "content": [
-    "wwwroot",
-    "web.config"
-  ],
-
-  "exclude": [
-    "node_modules"
-  ],
+  "publishOptions": {
+    "include": [
+      "wwwroot",
+      "web.config"
+    ],
+    "exclude": [ "node_modules" ]
```

Comment:
The `"exclude"` shouldn't be needed at all. You will however need to include the `appsettings.json`